samples/sampler/stories/default/BodyText.js

```diff
@@ -18,6 +18,7 @@ storiesOf('Agate', module)
 		() => (
 			<BodyText
 				centered={boolean('centered', Config)}
+				disabled={boolean('disabled', Config)}
```

Comment:
Why this prop is needed for BodyText?

Reply:
I think we can remove this. Please check it when merge.

Reply:
You are right, it is not in the GUI. Thanks!